### ISSUE TITLE
[Minor/Graphical] Top Pixel/Half Pixel of editbox is obscured

### ISSUE THREAD

**Issue by emptyrivers** [WeakAuras/WeakAuras2#261]
http://i.imgur.com/NjCohnI.png It's only really noticeable with square brackets, and only on certain themes. By editbox, I mean the one opened by `WeakAuras.OpenTextEditor(...)`.

Reproduced on latest alpha and release, as far back as #226, best as I can tell.

**Comment by emptyrivers** [comment]
On further inspection, this issue persists long before the previously referenced pull request, and changing the UI scale can cause it to vanish altogether, or worsen. I suspect that it's a pixel 'imperfection' issue.

**Comment by InfusOnWoW** [comment]
I'm pretty sure that's a Blizzard bug. And that probably means we can't do much about it.

I'll see if I can rerpoduce it outside of WA and whether it's font dependant.

**Comment by emptyrivers** [comment]
Using a font with a slightly shorter set of [square brackets] would probably help. Those 2 characters are taller than other characters in the default font of WeakAuras.

**Comment by InfusOnWoW** [comment]
So, I looked into it for more than a hour.

I could reproduce it, and it felt like a Blizzard bug. Adding more brackets sometimes made the text jump one pixel. After the jump it looked perfect.

I tried reproducing that in a smaller code example, but couldn't. It seemed to me that disabling the IndentationLib fixed it, but then I enabled it again and now I can't reproduce it anymore....

The indentation lib does nothing to the texteditor frame, it only adds a few color codes to the text.

So... That feels like enough wasted time. Unless @Stanzilla disagrees, I would just close this as "too obscure"
